catalyst/contrib/dl/callbacks/tests/test_cmc_metric_callback.py

```diff
@@ -0,0 +1,47 @@
+import torch
+from catalyst.contrib.dl.callbacks.cmc_callback import\
+    (CMCScoreCallback, _cmc_score_count)
+
+
+def test_metric_simple_0():
+    """simple test with two examples"""
+    distances = torch.tensor([[1, 2], [2, 1]])
```

Comment:
Could you reformat it for better readability in the following way?

distances = torch.tensor([
    [1, 2],
    [2, 1]
])

(the same thing for the code below)